apps/web/app/dashboard/search/page.tsx

```diff
@@ -1,35 +0,0 @@
-"use client";
```

Comment:
The search bar doesn't exist in all pages (e.g. in favourites, etc). I'd still keep the search page tab for quick search access.